en/docs/guides/usecases/order-management-automation.md

```
@@ -0,0 +1,429 @@
+
+3. Confirm the change landed in the database:
+
+    ```sql
+    SELECT order_id, status FROM orders_db.orders WHERE order_id IN ('ORD-001', 'ORD-002');
+    ```
+
+    Both rows now read `PROCESSING`.
+
+4. Run the automation a second time. Nothing is waiting, so the early exit kicks in and the terminal shows a single line:
+
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)